src/Resources/skeleton/event/Subscriber.tpl.php

```diff
@@ -0,0 +1,22 @@
+<?= '<?php' ?>
+
+
+namespace App\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+<?= $eventUseStatement ?>
+
```

Comment:
I think you need a second empty line here: The first EOL char will be swallowed by `?>`, so this would not create an empty line between the use statement and the class at the end.